.gitignore

```diff
@@ -1,3 +1,5 @@
 .env
 node_modules
-dist
\ No newline at end of file
+dist
+package-lock.json
+package.json
```

Comment:
Those probably shouldn't be in .gitignore because we might need to change e.g. some scripts in package.json